src/App.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import { developer, developerVanya } from './developer';
```

Comment:
```suggestion
import { Developer, DeveloperVanya } from './developer';
```
это же классы, поэтому лучше на это намекнуть:)